resources/definitions/HardPrint_Hozo.def.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Could you change the file name, and IDs referring to it, to lowercase like the rest of the printer definitions?

Reply:
ok, I fixed it.